testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -124,6 +124,32 @@ impl EmitJobRequest {
             },
         }
     }
+
+    pub fn fixed_tps_params(instance_count: usize, tps: u64) -> (usize, u64) {
+        let tps = if tps < 1 { 1 } else { tps };
+        let mut num_workers: usize = 10;
+        while instance_count * num_workers > tps as usize {
```

Comment:
I think there was some misunderstanding with my comment, i did not mint to `num_workers /= 2` every loop
I think this entire loop seems like an attempt to divide `tps / (instance_count * num_workers)` (possibly some +-1 here and there)